initial_posture: [0.0, 0.4, 1.3, 0.0, 0.0]
joint_waypoint_: [-0.1, 0.55, 0.7, 0.01, 0.29]
cartesian_waypoint_: [0.0, 0.0, -0.03, 0.0, 0.0, 0.0]
